src/os/fields/fields.js

```diff
@@ -8,6 +8,7 @@ goog.provide('os.fields');
 os.Fields = {
   ALT: 'ALTITUDE',
   ALT_UNITS: 'ALTITUDE_UNITS',
+  ALTITUDE_INPUT: 'ALTITUDE_INPUT',
```

Comment:
The user altitude input will get saved to the geometry, and doesn't need to be saved again as a property on the feature. You can remove this field and all references to it.